src/core/plugins/disableAuthorize.js

```diff
@@ -0,0 +1,9 @@
+// OutSystems change: create a plugin that disables the Authorize button 
+const DisableAuthorizePlugin = function() {
```

Comment:
This could be a lambda instead: `() => {`

Reply:
done